recipes/cuik-molmaker/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "cuik_molmaker" %}
+{% set version = "0.2" %}
+{% set rdkit_version = rdkit %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker/archive/refs/tags/v0.2.tar.gz
+  sha256: c137442f06af5f195efe7a916aefae065c8c52b40ae8e028a5bcd672b75c5fe8
+
+build:
+  script_env:
+    - RDKIT_VERSION={{ rdkit_version }}
+  number: 0
+  missing_dso_whitelist:  # [win]
+    - $RPATH/cuik_molmaker_core.dll  # [win]
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - cmake  # using CMake
+    - make   # [unix]
+    - ninja  # [win]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - python
+  host:
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - python
+    - numpy >=1.23
+    - setuptools >=80.0.0
+    - pip
+    - eigen
+    - librdkit-dev {{ rdkit_version }}
+    - rdkit {{ rdkit_version }}
+    - pybind11 >=2.13
+    - libboost-devel
+    - libboost-python-devel
+  run:
+    - python
+    - setuptools >=80.0.0
+    - librdkit {{ rdkit_version }}
+    - rdkit {{ rdkit_version }}
+
+test:
+  imports:
+    - cuik_molmaker
+  requires:
+    - pip
+    - pytest
+    - scipy
+    - numpy
+  source_files:
+    - tests/python/
+    - tests/data/
+  commands:
+    - pip check
+    - python -c "import cuik_molmaker; print('Listing all atom onehot features:');  print(cuik_molmaker.list_all_atom_onehot_features())"
+    - pytest -s tests/python/
+
+about:
+  home: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker
+  summary: 'C++/python module for featurizing molecules'
+  description: |
+    cuik-molmaker is a specialized package designed for molecular featurization,
+    converting chemical structures into formats that can be effectively used as 
+    inputs for deep learning models, particularly graph neural networks (GNNs).
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE/License.txt
+  license_url: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker/blob/main/LICENSE/License.txt
+  doc_url: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker/blob/main/README.md
+  dev_url: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker
+
+extra:
+  recipe-maintainers:
+    - sveccham
```

```suggestion
extra:
  feedstock-name: cuik-molmaker
  recipe-maintainers:
    - sveccham
```

Use this extra conda-forge-specific key to choose the exact name of the feedstock for this package.